openbb_platform_pro_backend/main.py

```diff
@@ -99,25 +117,130 @@ async def get_widgets():
 # pylint: disable=import-outside-toplevel
 def launch_api():
     """Main function."""
-    import os
     import uvicorn
 
+    if Path(CURRENT_USER_SETTINGS).exists():
+        with open(CURRENT_USER_SETTINGS, "r") as f:
+            current_settings = json.load(f)
+    else:
+        current_settings = {"credentials": {}, "preferences": {}, "defaults": {"commands": {}}}
+
+    pat = input(
```

Comment:
can you check if we can make the input be not fully visible by default? either not visible at all, like we usually see on linux when we enter the sudo password, or replacing each symbol with `*`

Reply:
Built-in methods allow for it not being visible at all.